ExcelWakaTimeAddin/WakaTimeAddIn.cs

```diff
@@ -69,7 +69,7 @@ private static void ApplicationOnWindowActivate(Excel.Workbook wb, Excel.Window
         {
             try
             {
-                WakaTime.HandleActivity(wb.FullName, false, string.Empty);
+                WakaTime.HandleActivity(GetFileLocalPath(wb.FullName), false, string.Empty);
```

Comment:
Let's call `GetFileLocalPath` in the `HandleActivity` so it's called in one place instead of multiple.